client/client.js

```diff
@@ -405,3 +405,20 @@ function expressMetricsEndpointDetail(modelOrUri, windowStartTime, callback) {
   );
 }
 Client.prototype.expressMetricsEndpointDetail = expressMetricsEndpointDetail;
+
+function executorCreate(driver, callback) {
```

Comment:
@rmg I am not sure what parameters will go in here. I would prefer to evolve this function as we expand on functionality in mesh. Im sure it will look different by the time its time for out first release